core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -865,10 +835,6 @@
           + " table.summarizer.<unique id>.opt.<key>=<value>."),
 
   // VFS ClassLoader properties
-  VFS_CLASSLOADER_SYSTEM_CLASSPATH_PROPERTY(
```

Comment:
I need to check, but I think this one is probably still in use via `AccumuloVFSClassLoader.VFS_CLASSLOADER_SYSTEM_CLASSPATH_PROPERTY` and exists here so it will get put in the documentation.

Reply:
Ah I see how I missed those usages. Fixing in new commit